doc/man/scc-hypervisor-collector.1.md

```diff
@@ -0,0 +1,203 @@
+---
+title: SCC-HYPERVISOR-COLLECTOR
+section: 1
+header: SCC Hypervisor Collector
+date: June 2022
+---
+# NAME
+**scc-hypervisor-collector** - Collect & Upload Hypervisor details
+to SUSE Customer Center
+
+# SYNOPSIS
+
+**scc-hypervisor-collector [<optional>...]**
+
+# DESCRIPTION
+
+The **scc-hypervisor-collector** collects details that are relevant
+to SUSE Customer Center (SCC) subscription compliance tracking, from
+the specified hypervisors.
+
+The collected details include the total system RAM, CPU architecture
+and topology of the hypervisor nodes themselves, as well as the UUIDs
+and state (running or shutoff) for the hosted VMs.
+
+## HYPERVISOR DETAILS RETRIEVAL
+
+The **gatherer** Python module provided by the **virtual-host-gatherer(1)**
+command is used to retrieve the details from the configured hypervisors.
+
+# OPTIONS
+
+  **-c**, **--config <CONFIG_FILE>**
+  : Specifies a YAML file containing configuration settings.
+
+  **--config_dir**, **--config-dir <CONFIG_DIR>**
+  : Specifies a directory containing YAML configuration files that
```

Comment:
this looks right here, but the line wrapping in my rendered copy has words hanging to the next line, may need to check that